src/main/resources/application.yml

```
@@ -20,4 +20,4 @@ spring:
     properties:
       hibernate:
         format_sql: true
-        use_sql_comments: true
\ No newline at end of file
+        use_sql_comments: tru
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)